verrazzano/verrazzano
# VZ-10130 Add the cluster resource utilization dashboard

**Changes**
- Created a cluster resource utilization dashboard with Used, Max, and Percentage values for:
- CPU, Memory, and Storage

**Testing**
- Install a KinD multicluster and OKE cluster
- Verify that for each cluster, the metrics accurately reflect the cluster values
